addons/website_slides/controllers/main.py

```diff
@@ -537,6 +537,10 @@ def slide_view(self, slide, **kwargs):
             'search_uncategorized': kwargs.get('search_uncategorized')
         })
 
+        values = self._prepare_additional_channel_values(values, slide.channel_id, **kwargs)
```

Comment:
To complete the other comment: you could force channel in values here and avoid adding a new parameter I think.